playbooks/nvidia-gpu-driver.yml

```diff
@@ -1,19 +1,27 @@
 ---
-- hosts: gpu-servers
+- hosts: all
   become: true
   become_method: sudo
+  vars:
+    nv_gpu_apt_repo_url: ppa:graphics-drivers/ppa
   tasks:
+    - name: Check for GPUs
```

Comment:
@dholt why was this useful? Do you want me to add it to the new, refactored playbook+role?